code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -66,62 +66,100 @@
 /obj/structure/closet/secure_closet/CSECO
 	name = "chief security officer's locker"
 	req_access = list(access_armory)
-	icon_state = "securenew1"
-	icon_closed = "securenew"
-	icon_locked = "securenew1"
+	icon_state = "secure1"
+	icon_closed = "secure"
+	icon_locked = "secure1"
 	icon_opened = "secureopen"
-	icon_off = "securenewoff"
+	icon_off = "secureoff"
 
 /obj/structure/closet/secure_closet/CSECO/WillContain()
 	return list(
 		new/datum/atom_creator/weighted(list(/obj/item/weapon/storage/backpack/security, /obj/item/weapon/storage/backpack/satchel_sec)),
 		new/datum/atom_creator/simple(/obj/item/weapon/storage/backpack/dufflebag/sec, 50),
 		/obj/item/clothing/under/deadspace/security,
-		/obj/item/clothing/shoes/ds_securityboots,
+		/obj/item/clothing/head/helmet/pcsi,
+		/obj/item/clothing/suit/armor/vest/pcsi,
+		/obj/item/clothing/gloves/combat/pcsi,
+		/obj/item/clothing/shoes/pcsi,
+		/obj/item/clothing/glasses/hud/security,
+		/obj/item/clothing/mask/gas,
 		/obj/item/device/radio/headset/heads/cseco,
-		/obj/item/clothing/glasses/sunglasses/sechud,
+		/obj/item/weapon/storage/belt/holster/security,
 		/obj/item/weapon/gun/projectile/divet,
 		/obj/item/ammo_magazine/divet = 2,
+		/obj/item/weapon/reagent_containers/spray/pepper,
+		/obj/item/weapon/storage/box/holobadge,
+		/obj/item/weapon/melee/baton/loaded,
 		/obj/item/taperoll/police,
+		/obj/item/device/hailer,
+		/obj/item/device/holowarrant,
+		/obj/item/device/flashlight/maglight
+	)
+
+/obj/structure/closet/secure_closet/SSO
+	name = "senior security officer's locker"
+	req_access = list(access_security)
+	icon_state = "secure1"
+	icon_closed = "secure"
+	icon_locked = "secure1"
+	icon_opened = "secureopen"
+	icon_off = "secureoff"
+
+/obj/structure/closet/secure_closet/SSO/WillContain()
+	return list(
+		new /datum/atom_creator/weighted(list(/obj/item/weapon/storage/backpack/security, /obj/item/weapon/storage/backpack/satchel_sec)),
+		new /datum/atom_creator/weighted(list(/obj/item/weapon/storage/backpack/dufflebag/sec, /obj/item/weapon/storage/backpack/messenger/sec)),
+		/obj/item/clothing/under/deadspace/security,
+		/obj/item/clothing/head/helmet/pcsi,
+		/obj/item/clothing/suit/armor/vest/pcsi,
+		/obj/item/clothing/gloves/combat/pcsi,
+		/obj/item/clothing/shoes/pcsi,
+		/obj/item/clothing/gloves/forensic,
+		/obj/item/clothing/glasses/hud/security,
 		/obj/item/clothing/mask/gas,
-		/obj/item/device/flashlight/maglight,
-		/obj/item/weapon/storage/box/teargas,
+		/obj/item/device/radio/headset/headset_sec,
+		/obj/item/weapon/storage/belt/holster/forensic,
 		/obj/item/weapon/storage/belt/holster/security,
+		/obj/item/weapon/gun/projectile/divet,
+		/obj/item/ammo_magazine/divet = 2,
 		/obj/item/weapon/reagent_containers/spray/pepper,
 		/obj/item/weapon/melee/baton/loaded,
-		/obj/item/weapon/storage/box/holobadge,
+		/obj/item/taperoll/police,
+		/obj/item/device/hailer,
 		/obj/item/device/holowarrant,
-		/obj/random/tool,
-		/obj/item/weapon/rig/riot
```

Comment:
Have you discussed this with west? The placement of the riot rig there was kind of temporary, but its future role hasn't been entirely determined

Reply:
Yes.